msfdb

```diff
@@ -641,16 +641,34 @@ end
 def output_web_service_information
   puts ''
   puts 'MSF web service configuration complete'
-  puts 'Add the data service in msfconsole using the command:'
-  # build data services command based on install options
-  ds_cmd = "data_services --add --token #{@ws_api_token}"
-  ds_cmd << " --ssl --cert #{@options[:ssl_cert]}" if @options[:ssl]
-  ds_cmd << " --skip-verify" if skip_ssl_verify?
-  ds_cmd << " #{get_web_service_host}"
-  puts "#{ds_cmd}"
+  puts 'Connect to the data service in msfconsole using the command:'
+  # build db_connect command based on install options
+  connect_cmd = "db_connect --token #{@ws_api_token}"
+  connect_cmd << " --cert #{@options[:ssl_cert]}" if @options[:ssl]
+  connect_cmd << " --skip-verify" if skip_ssl_verify?
+  connect_cmd << " #{get_web_service_uri}"
+  puts "#{connect_cmd}"
   puts ''
   puts 'The username and password are credentials for the API account:'
   puts "#{get_web_service_uri(path: '/api/v1/auth/account')}"
+  puts ''
+
+  persist_data_service(connect_cmd: connect_cmd)
+end
+
+def persist_data_service(connect_cmd:)
+  if ask_yn('Add data service connection to local msfconsole and persist as default?')
+    data_service_name = "local-#{@options[:ssl] ? 'https' : 'http'}-data-service"
+    data_service_name = ask_value('Data service connection name?', data_service_name)
+    # execute msfconsole commands to add and persist the data service connection
+    cmd = "msfconsole -qx \"#{connect_cmd}; db_save -d #{data_service_name}; exit\""
+    if run_cmd(cmd) != 0
```

Comment:
This actually takes a decent amount of time (10ish seconds) to execute because it has to start up framework. We could skip it by just writing out the values to the config file. However, I do like this method because it keeps the formatting up to date if we make any changes to the config file. 

Not sure we need a change here but I did want to point this out.